Server/src/main/java/app/ModelController.java

```diff
@@ -124,12 +123,31 @@ public void checkModels() {
                 .stream()
                 .filter(tm -> tm.getStatusID() == Constants.STATUS_LIVE)
                 .forEach(this::trainAndSave);
+        trainedModelsToRetrain
+                .stream()
+                .filter(tm -> tm.getStatusID() == Constants.STATUS_EVALUATION)
+                .findFirst().ifPresent(this::evaluate);
         trainedModelsToRetrain
                 .stream()
                 .filter(tm -> tm.getStatusID() == Constants.STATUS_EVALUATION)
                 .findFirst().ifPresent(this::trainAndSave);
     }
```

Comment:
you cant do that....like i told you, simply add an if statement in trainAndSave after the model is trained to check if the statusID is Eval and then do the evaluate

Reply:
removed this code